client/proposer/service_test.go

```diff
@@ -119,35 +145,20 @@ func TestIncompleteCollation(t *testing.T) {
 		<-fakeProposer.msgChan
```

Comment:
Remove the receive from the channel here too

Reply:
Done.